packages/web-components/tests/e2e-storybook/cypress.json

```diff
@@ -14,8 +14,8 @@
   "screenshotOnRunFailure": false,
   "video": false,
   "testFiles": "**/*.e2e.js",
-  "pageLoadTimeout": 90000,
-  "defaultCommandTimeout": 90000,
+  "pageLoadTimeout": 30000,
+  "defaultCommandTimeout": 4000,
```

Comment:
This represents a big change in terms of the performance of our tests. See https://docs.cypress.io/app/references/configuration#Timeouts. The values of 90000 previously meant that errant tests were very slow. Especially for `defaultCommandTimeout`, if say a selector failed to match due to an issue with the test, each time we'd look for that selector, the test would stall for 1.5min before failing and moving on.